### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by SeanFeldman**
Some cleanups that are possible with C# 7

### CODE REVIEW

**Comment by SeanFeldman** [comment]
@jtaubensee targeting your branch so that you can pick it up and continue if you like it.

File: src/Microsoft.Azure.ServiceBus/MessagingUtilities.cs [hunk 1]
```diff
@@ -25,8 +25,8 @@ public static TimeSpan CalculateRenewAfterDuration(DateTime lockedUntilUtc)
 
         public static bool ShouldRetry(Exception exception)
         {
-            ServiceBusException serviceBusException = exception as ServiceBusException;
-            return serviceBusException != null && serviceBusException.IsTransient;
+            var serviceBusException = exception as ServiceBusException;
+            return serviceBusException?.IsTransient == true;
```
Comment (nemakam) [inline src/Microsoft.Azure.ServiceBus/MessagingUtilities.cs]: >?.IsTransient == true; [](start = 38, length = 22)

Is `== true` required?

**Comment by nemakam** [comment]
:shipit:

File: src/Microsoft.Azure.ServiceBus/MessagingUtilities.cs (hunk 1 above)
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/MessagingUtilities.cs]: Just adds a bit of clarity, but definitely not required.

**Pull request merged**